Doc/library/collections.abc.rst

```diff
@@ -185,14 +184,8 @@ ABC                            Inherits from          Abstract Methods        Mi
 .. [1] These ABCs override :meth:`object.__subclasshook__` to support
    testing an interface by verifying the required methods are present
    and have not been set to :const:`None`.  This only works for simple
-   interfaces.  More complex interfaces require registration or direct
-   subclassing.
-
-.. [2] Checking ``isinstance(obj, Iterable)`` detects classes that are
```

Comment:
I would like to leave this footnote as is.   While it is a bit redundant, it is an essential point and addressed a common point of confusion.

Reply:
I see, I restored it.